packages/shell-evaluator/src/shell-evaluator.ts

```diff
@@ -25,7 +25,9 @@ try {
 }
 if (v8?.startupSnapshot?.isBuildingSnapshot?.()) {
   v8.startupSnapshot.addSerializeCallback(() => {
+    // Ensure that any lazy loading performed by Babel is part of the snapshot
     eval(new AsyncWriter().runtimeSupportCode());
+    eval(new AsyncWriter().process('1+1'));
```

Comment:
Just to make sure I understand, we are forcing the evaluation of code so we ensure that babel runs and so the async rewriter set up?

Reply:
Yup, that's exactly it 👍